src/nft/nftId.ts

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'crypto';
+import { bech32 } from 'bech32';
+import { getISCNPrefix } from '../iscn/parsing';
+
+export function calculateNFTClassIdByAccount(
+  account: string,
+  numberOfExistingNFTClass = 0,
+): string {
+  let bytes = Buffer.from(account, 'utf8');
+  bytes = Buffer.concat([
+    bytes,
+    Buffer.from(numberOfExistingNFTClass.toString(), 'utf8'),
+  ]);
+  const sha256 = createHash('sha256');
+  const hash = sha256.update(bytes).digest();
+  return bech32.encode('likenft', bech32.toWords(hash));
+}
+
+export function calculateNFTClassIdByISCNId(
+  iscnId: string,
+  numberOfExistingNFTClass = 0,
+): string {
+  const iscnIdPrefix = getISCNPrefix(iscnId);
+  return calculateNFTClassIdByAccount(iscnIdPrefix, numberOfExistingNFTClass);
+}
```

Comment:
```suggestion
export function calculateNFTClassIdByAccount(
  prefix: Uint8Array,
  serial = 0,
): string {
  const bytes = Buffer.concat([
    prefix,
    Buffer.from(serial.toString(), 'utf8'),
  ]);
  const sha256 = createHash('sha256');
  const hash = sha256.update(bytes).digest();
  return bech32.encode('likenft', bech32.toWords(hash));
}

export function calculateNFTClassIdByISCNId(
  iscnId: string,
  numberOfExistingNFTClass = 0,
): string {
  const iscnIdPrefix = getISCNPrefix(iscnId);
  const prefix = Buffer.from(iscnIdPrefix, 'utf8');
  return calculateNFTClassIdByAccount(prefix, numberOfExistingNFTClass);
}
```

Reply:
Was working on my version before you post this suggestion.. Updated

Reply:
Should we rename `calculateNFTClassIdByAccount` and don't export it instead?